tests/ui/test_implementation_page.py

```
@@ -96,37 +111,179 @@ def test_format_options_structure(self):
+            assert hasattr(generator, "chain")
+
+            # Test template functionality with SlideTemplate
+            from pathlib import Path
+
+            from src.models import SlideTemplate
+
+            template = SlideTemplate(
+                id="test_template",
+                name="Test Template",
+                description="Test template for integration tests",
+                template_dir=Path("/tmp/test_template"),
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/test_template"

(S108)